lib/ruby_lsp/requests/document_link.rb

```diff
@@ -63,6 +101,61 @@ def gem_paths
             lookup
           end, T.nilable(T::Hash[String, T::Array[String]]))
         end
+
+        sig { returns(T.nilable(T::Hash[String, T::Hash[Symbol, String]])) }
+        def rails_documents
+          @rails_documents ||= T.let(begin
+            return {} unless RAILTIES_VERSION
+
+            $stderr.puts "Fetching Rails Documents..."
+            response = Net::HTTP.get_response(URI("#{RAILS_DOC_HOST}/v7.1.1/js/search_index.js"))
```

Comment:
Should the version here be `RAILTIES_VERSION`?

Reply:
I was testing something and accidentally committed it 🤦